src/bbs-plus/encoder.ts

```diff
@@ -41,6 +41,9 @@ export class Encoder {
   encodeMessage(name: string, value: unknown, strict = false): Uint8Array {
     const encoder = this.encoders?.get(name) || this.defaultEncoder;
     if (encoder !== undefined) {
+      if (typeof value === undefined) {
```

Comment:
A check for this condition [here](https://github.com/docknetwork/crypto-wasm-ts/blob/master/tests/composite-proofs/msg-js-obj/util.spec.ts#L68) would be nice.

Reply:
added